### ISSUE TITLE
It is allowed to include one Object into the Storage Group multiple times

### ISSUE THREAD

**Issue by aprasolova** [nspcc-dev/neofs-node#1372]
## Expected Behavior
When I try to create a Storage Group that holds an OID multiple times, I get warning message and the request isn't sent.

## Current Behavior
I'm able to create an SG with the same OID repeated in `--members`
```
$ neofs-cli --rpc-endpoint s01.neofs.devenv:8080 --wallet wallet.json --config wallet_pass.yml storagegroup put \
--cid HJGELpCdrWnadKsiLCjrkbT1rSsBgVmjQuS3V6sDUoLG \
--members ErXBo7atAaCEibvggpYNmHMNWyBszFFnXwYhLygST4b6,ErXBo7atAaCEibvggpYNmHMNWyBszFFnXwYhLygST4b6

 Storage group successfully stored
  ID: G8rqK2UeWCTATNwGucKSUD3GGX3NWnALQhAmTuAq9BW4
  CID: HJGELpCdrWnadKsiLCjrkbT1rSsBgVmjQuS3V6sDUoLG
```

## Steps to Reproduce (for bugs)
1. Create a container
2. Put an object to that container
3. Create SG with OID repeated in `--members` key several times

## Your Environment
```
$ neofs-cli --version
Version: v0.28.1 
Build: 2022-05-05T09:07:45+00:00 
Debug: false
```

**Comment by cthulhu-rider** [comment]
* track https://github.com/nspcc-dev/neofs-api/issues/218

**Comment by fyrchik** [comment]
Fixed via #1490 .
